user_guide/intro_users_and_benefits.rst

```diff
@@ -0,0 +1,163 @@
+.. _oro-benefits:
+
+OroCRM Users and Their Benefits
+===============================
+
+This chapter of the guide explains the most common OroCRM user roles in order to help you understand which sections of 
+this user guide best match your work duties.
+
+Something for Everyone
+----------------------
+
+Collect and Process information easily
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+OroCRM handles critical CRM tasks such as:
+
+- Collect and aggregate information related to customers, sales, marketing campaigns, sales pipeline, and high level 
+  business performance. 
+
+- Organize the information in a way that is both meaningful and efficient
+
+- Ensure easy, convenient, and safe access to information for all the stakeholders with customizable access and 
+  permission settings. 
+
+- Create workflows that mirror or improve upon existing business processes as well as provide an overhead view of that 
+  pipeline  
```

Comment:
Edit: - Creating workflows that mirror or improve upon existing business processes as well as providing an overhead view of that pipeline.